cfgov/apache/conf.d/newsroom.conf

```diff
@@ -0,0 +1,12 @@
+SetEnv NEWSROOM enable
```

Comment:
This newsroom.conf file is obsolete and has been removed from the internal repository; it should be removed here as well.